debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -415,6 +415,8 @@ J9ROMClass.doubleScalarStaticCount = required
 J9ROMClass.extraModifiers = required
 J9ROMClass.innerClassCount = required
 J9ROMClass.innerClasses = required
+J9ROMClass.enclosedInnerClassCount = required
+J9ROMClass.enclosedInnerClasses = required
```

Comment:
Removed as suggested.

Reply:
They can't be removed either, otherwise `RomClassWalker` will not be able to make references to those fields (the rule for `BUILD_J9DDR_TEST_CLASSES` will fail). The additions must specify types to be used for old core dumps:
```
J9ROMClass.enclosedInnerClassCount = U_32
J9ROMClass.enclosedInnerClasses = J9SRP(J9SRP(J9UTF8))
```

Reply:
Agreed and fixed as suggested.